tap_quickbooks/schemas/shared/group_line.json

```diff
@@ -45,7 +45,8 @@
                 }
               },
               "Qty": {
-                "type": ["null", "integer"]
+                "format": "singer.decimal",
```

Comment:
Same of Qty if it is quantity then it won't be in decimal.

Reply:
Reverted back changes for Qty.